core/src/main/java/hudson/tasks/ArtifactArchiver.java

```diff
@@ -319,16 +359,57 @@ public String getDisplayName() {
          * Performs on-the-fly validation of the file mask wildcard, when the artifacts
          * textbox or the caseSensitive checkbox are modified
          */
-        public FormValidation doCheckArtifacts(@AncestorInPath AbstractProject project,
-                @QueryParameter String value,
+        public FormValidation doCheckArtifacts(@AncestorInPath AbstractProject project, 
```

Comment:
If you are going to change the signature, at least take the opportunity to restrict this, as it should have been to begin with.